style.css

```diff
@@ -0,0 +1,110 @@
+.wrapper {
+  width: 80%;
+  margin: auto;
+  border: dashed 1px red;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__logo {
+  padding: 40px 0;
+  font-family: 'ptserif-bold', arial, helvetica, Sans-serif;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 31px;
+  position: relative;
+  padding-left: 44px;
+}
+
+.logo__circle {
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)